ChatQnA/docker/chatqna_no_wrapper.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import re
+
+from comps import ChatQnAGateway, MicroService, ServiceOrchestrator, ServiceType
+
+
+class ChatTemplate:
```

Comment:
how to customize template by users

Reply:
I think here we move the logics here in mega definition to let users customize the template themselves